Index: B-11-Cấu trúc lặp 2/Bài tập/Luyện tập vòng lặp 02/Luyện tập vòng lặp 02-Bài 5.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/B-11-Cấu trúc lặp 2/Bài tập/Luyện tập vòng lặp 02/Luyện tập vòng lặp 02-Bài 5.html b/B-11-Cấu trúc lặp 2/Bài tập/Luyện tập vòng lặp 02/Luyện tập vòng lặp 02-Bài 5.html
new file mode 100644
--- /dev/null	
+++ b/B-11-Cấu trúc lặp 2/Bài tập/Luyện tập vòng lặp 02/Luyện tập vòng lặp 02-Bài 5.html	
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Luyện tập vòng lặp 02-Bài 5</title>
+</head>
+<body>
+<script>
+    let value = parseInt(prompt("Nhập số tiền !!!"));
+    let a = parseInt(prompt("Nhập số tháng !!!"));
+    let count = 0
+    let result = 0
+    let total = 0
+    for (let i = 0; i < value; i++) {
+        result = (value * 2) / 100;
+        count = a * result;
+        total = value + count;
+
+    }
+    document.write("Tiền lãi 1 tháng : " + result + "<br>");
+    document.write("Tiền lãi " + a + " tháng : " + count + "<br>");
+    document.write("Tổng số tiền cần phải thanh toán : " + total);
+</script>
+</body>
+</html>
\ No newline at end of file
Index: B-11-Cấu trúc lặp 2/Bài tập/Luyện tập vòng lặp 02/Luyện tập vòng lặp 02-Bài 4.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/B-11-Cấu trúc lặp 2/Bài tập/Luyện tập vòng lặp 02/Luyện tập vòng lặp 02-Bài 4.html b/B-11-Cấu trúc lặp 2/Bài tập/Luyện tập vòng lặp 02/Luyện tập vòng lặp 02-Bài 4.html
new file mode 100644
--- /dev/null	
+++ b/B-11-Cấu trúc lặp 2/Bài tập/Luyện tập vòng lặp 02/Luyện tập vòng lặp 02-Bài 4.html	
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Luyện tập vòng lặp 02-Bài 4</title>
+</head>
+<body>
+<script>
+    for (let i = 0; i <= 7; i++) {
+        for (let j = 0; j < 20; j++) {
+            if (i === 0 || i === 7 || j === 0 || j ===19) {
+                document.write("*");
+            } else {
+                document.write("&ensp;");
+            }
+        }
+        document.write("<br>");
+    }
+
+</script>
+</body>
+</html>
\ No newline at end of file
